docs/anonymized-data.md

```diff
@@ -0,0 +1,82 @@
+# Anonymized Data
+
+This is an approximate list of data point (Kubernetes/OpenShift resource fields)
+that are anonymized by the Insights Operator. This list is subject to change and
+may not include all anonymized fields.
```

Comment:
Is it worth explaining why this list could be incomplete? The wording used here leaves some open questions